packages/conf-g++/conf-g++.1.0/opam

```diff
@@ -23,6 +23,7 @@ depexts: [
   ["gcc-g++"] {os = "win32" & os-distribution = "cygwinports"}
   ["gcc"] {os = "freebsd"}
 ]
+available: os != "openbsd"
```

Comment:
```suggestion
  ["gcc"] {os = "freebsd" | os = "openbsd"}
]
```

I'm not sure because I don't have access to an openbsd machine. But the internet seems to suggest this.

Reply:
merging this and might have to correct later